docs/src/main/asciidoc/health-guide.adoc

```diff
@@ -56,141 +58,171 @@ which is an implementation of the MicroProfile Health specification used in Quar
+== Adding a readiness health check procedure
 
-In previous section we saw how to create a simple health check with only the minimal 
-attributes, namely, the health check name and its state (UP or DOWN). However, the 
-MicroProfile specification also provides a way for the applications to supply 
-arbitrary data in the form of key value pairs sent to the consuming end. This can be done by using the 
-`withData(key, value)` method of the health check response builder API.
+In the previous section, we created a simple liveness health check procedure which states 
+whether our application is running or not. In this section, we will create a readiness 
+health check which will be able to state whether our application is able to process 
+requests.
 
-Let's create our second health check procedure `org.acme.health.DataHealthCheck`:
+We will create another health check procedure that simulates a connection to 
+an external service provider such as a database. For starters, we will always return 
+the response indicating the application is ready.
+
+Create `org.acme.health.DatabaseConnectionHealthCheck` class:
 
 [source,java]
 ----
 package org.acme.health;
 
-import org.eclipse.microprofile.health.Health;
 import org.eclipse.microprofile.health.HealthCheck;
 import org.eclipse.microprofile.health.HealthCheckResponse;
+import org.eclipse.microprofile.health.Readiness;
 
 import javax.enterprise.context.ApplicationScoped;
 
-@Health
+@Readiness
 @ApplicationScoped
-public class DataHealthCheck implements HealthCheck {
+public class DatabaseConnectionHealthCheck implements HealthCheck {
 
     @Override
     public HealthCheckResponse call() {
-        return HealthCheckResponse.named("Health check with data")
-                .up()
-                .withData("foo", "fooValue")
-                .withData("bar", "barValue")
-                .build();
+        return HealthCheckResponse.named("Database connection health check").up().build();
     }
 }
+
 ----
 
-If you rerun the health check procedure again by accessing the `/health` endpoint you can 
-see that the new health check `Health check with data` is present in the `checks` array. 
-This check contains a new attribute called `data` which is a JSON object consisting of 
-the properties we have defined in our health check procedure.
+If you now rerun the health check at `http://localhost:8080/health/live` the `checks` 
+array will contain only previously defined `SimpleHealthCheck` as it is the only 
+defined check with the `@Liveness` qualifier. However, if you access 
+`http://localhost:8080/health/ready` (in the browser or by 
+`curl http://localhost:8080/health/ready`) you will see only the 
+`Database connection health check` as it is the only health check defined with the 
+`@Readiness` qualifier as the readiness health check procedure.
 
-== Negative health check procedure
+NOTE: If you access `http://localhost:8080/health` you will get back both checks.
```

Comment:
Same here, I would remove that.

Reply:
`/health` is still present in MP Health 2.0. There is a possibility that it will be extended in the future. I don't think it's deprecated so I would keep the note for now.